contracts/core/interfaces/ILiquidator.sol

```diff
@@ -97,33 +86,10 @@ interface ILiquidator {
     // Bidding Parameters
     // ----------------------------------------------------------------------
 
-    function minimumBid(address _set)
-        external
-        view
-        returns (uint256);
-
-    function remainingCurrentSets(address _set)
-        external
-        view
-        returns (uint256);
-
-    function getCombinedCurrentUnits(address _set)
-        external
-        view
-        returns (uint256[] memory);
-
-    function getCombinedNextSetUnits(address _set)
-        external
-        view
-        returns (uint256[] memory);
-
-    function getCombinedTokenArray(address _set)
-        external
-        view
-        returns (address[] memory);
-
-    function startingCurrentSetAmount(address _set)
-        external
-        view
-        returns (uint256);
+    function minimumBid(address _set) external view returns (uint256);
+    function remainingCurrentSets(address _set) external view returns (uint256);
+    function getCombinedCurrentUnits(address _set) external view returns (uint256[] memory);
+    function getCombinedNextSetUnits(address _set) external view returns (uint256[] memory);
+    function getCombinedTokenArray(address _set) external view returns (address[] memory);
+    function startingCurrentSetAmount(address _set) external view returns (uint256);
```

Comment:
Feels weird to have this one interface have the function sigs all on one line, since we aren't really doing that anywhere else.